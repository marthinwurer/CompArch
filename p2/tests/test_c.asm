;
; tests SEC/CLC
;
; uses instructions HALT
;
; uses no addressing modes
;
	.globl	start
start:	sec
	clc
	halt
	.end	start
