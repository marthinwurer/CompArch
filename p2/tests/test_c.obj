000 2 b1 00
002 2 a1 00
004 2 00 00
000
