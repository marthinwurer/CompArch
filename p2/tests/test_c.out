CPU "ARCH" Simulator, 2.5a(Jan 22 2015)
-----------------------------------------

Memory sets starting address to 0
000000:  PS[00]  000261  SEC
000002:  PS[01]  000241  CLC
000004:  PS[00]  000000  HALT

Machine Halted - HALT instruction

Simulated time 13 cycles

LAST CPUObject DESTROYED; END OF SIMULATION
